core/web/resolver/query.go

```diff
@@ -238,24 +244,25 @@ func (r *Resolver) Node(ctx context.Context, args struct{ ID graphql.ID }) (*Nod
 	r.App.GetLogger().Debug("resolver Node args %v", args)
 	name := string(args.ID)
 	r.App.GetLogger().Debug("resolver Node name %s", name)
-	node, err := r.App.EVMORM().NodeStatus(name)
-	if err != nil {
-		r.App.GetLogger().Errorw("resolver getting node status", "err", err)
 
-		if errors.Is(err, chains.ErrNotFound) {
-			npr, warn := NewNodePayloadResolver(nil, err)
-			if warn != nil {
-				r.App.GetLogger().Warnw("Error creating NodePayloadResolver", "name", name, "err", warn)
+	for _, relayer := range r.App.GetRelayers().Slice() {
+		statuses, _, _, err := relayer.ListNodeStatuses(ctx, 0, "")
+		if err != nil {
+			return nil, err
+		}
+		for i, s := range statuses {
+			if s.Name == name {
+				npr, err2 := NewNodePayloadResolver(&statuses[i], nil)
+				if err2 != nil {
+					return nil, err2
+				}
+				return npr, nil
 			}
-			return npr, nil
 		}
-		return nil, err
 	}
 
-	npr, warn := NewNodePayloadResolver(&node, nil)
-	if warn != nil {
-		r.App.GetLogger().Warnw("Error creating NodePayloadResolver", "name", name, "err", warn)
-	}
+	r.App.GetLogger().Errorw("resolver getting node status", "err", chains.ErrNotFound)
+	npr, _ := NewNodePayloadResolver(nil, chains.ErrNotFound)
```

Comment:
OK to ignore? If so, maybe leave a comment why.

Reply:
Fixed in https://github.com/smartcontractkit/chainlink/pull/11386/commits/9c0a2fe39e1447a40b9cc7bfd7489db2f396458f